lib/vagrant-openshift/command/test_origin.rb

```diff
@@ -74,6 +74,11 @@ def execute
             o.on("-r","--image-registry", String, "Image registry to configure tests with") do |f|
               options[:image_registry] = f
             end
+
+            o.on("", "--env ENV=VALUE", String, "Environment variable to execute tests with") do |f|
```

Comment:
You are passing 1 env at a time.  But the entry in options is storing all of the envs.